.github/ISSUE_TEMPLATE/data_request.md

```diff
@@ -0,0 +1,19 @@
+---
+name: Data request
+about: Suggest data to add to this platform
+
+---
+
+**Please describe the data required**
+A clear and concise description of the data needed
+
+**Where does the data fit in the country profiles already uploaded**
+A clear and concise description of the countries this data covers and the [topics](https://takwimu.africa/) under which this data would fall
+
+**Could you share possible sources for this data if any?**
```

Comment:
Reword - "Please share links to the sources for the data."

We should also mention allowing attaching of the file or link to gSheet here - https://blog.github.com/2015-09-25-attach-files-to-comments/